lib/services/mail_service.dart

```diff
@@ -0,0 +1,98 @@
+import 'dart:convert';
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:enough_mail/enough_mail.dart';
+import 'package:summer2022/exceptions/fetch_mail_exception.dart';
+import 'package:summer2022/models/Digest.dart';
+
+/// todo: remove this once mailpiece class is properly implemented
+class MailPiece {
+  String ID;
+  String EmailID;
+  DateTime TimeStamp = new DateTime(2022, 1, 1);
+  String Sender;
+  String MidID;
+  String ImageText;
+
+  MailPiece(this.ID, this.EmailID, this.MidID, this.ImageText, this.Sender,
+      this.TimeStamp);
+
+  factory MailPiece.fromJson(dynamic json) {
+    return MailPiece(
+        json['ID'] as String,
+        json['EmailID'] as String,
+        json['MidID'] as String,
+        json['ImageText'] as String,
+        json['Sender'] as String,
+        json['TimeStamp'] as DateTime);
+  }
+}
+
+class MailService {
+  /// Location of mail data file
+  String mailData = "";
+
+  Future<File> get _localFile async {
```

Comment:
This isn't really `async` and doesn't return a `Future`.
Also, it's probably not a good use case for a getter, since it's creating a new object. Instead, I'd set this in the constructor and keep it an instance variable.
See: https://dart.dev/guides/language/effective-dart/design#do-use-getters-for-operations-that-conceptually-access-properties